src/test/lambda/wizards/samInitWizard.test.ts

```diff
@@ -395,16 +447,17 @@ describe('CreateNewSamAppWizard', async function() {
                 })
             })
 
-            describe('location', async function() {
-                it('uses user response as schema', async function() {
+            describe('location', async function () {
```

Comment:
This spacing change is really distracting, can you run prettier on the whole repo as a separate PR we merge quickly?

Reply:
Out of scope for this PR.